wiki/Tournaments/CWC/1/en.md

```diff
@@ -5,149 +5,149 @@ tags:
   - [Shoichiro Sakamoto - Eye of Aeon (KanaRin) \[Kana's CTB\]](https://puu.sh/1g0e)
-  - [LEAF XCEED Music Division - Fur Elise \[Extra CtB Stage\]](https://puu.sh/HiC)
-  - [Yousei Teikoku - Kokou no Sousei \[Verdi's TAG2\]](https://puu.sh/f6oX)
-  - Ryu\* - I'm so Happy \[Shana's Madness CTB\]
+  - [LEAF XCEED Music Division - Fur Elise (Louis Cyphre) \[Extra CtB Stage\]](https://puu.sh/HiC)
+  - [Yousei Teikoku - Kokou no Sousei (Verdisphaena) \[Verdi's TAG2\]](https://puu.sh/f6oX)
+- Tiebreaker  
+  - **[Ryu\* - I'm so Happy (\[JPSNS\]Shana) \[Shana's Madness CTB\]](https://osu.ppy.sh/beatmapsets/30006#fruits/99319)**
 
 ### Semifinals
 
-- NoMod
+- NoMod (*to be played in order*)
   - [BACK-ON - flyaway \[Tales of\] Remix (ztrot) \[FLYAWAY!!!\]](https://osu.ppy.sh/beatmapsets/11620#fruits/44782)
   - [Demetori - Solar Sect of Mystic Wisdom \~ Nuclear Fusion (Louis Cyphre) \[Atomic\]](https://osu.ppy.sh/beatmapsets/20125#fruits/91037)
-  - [Basshunter - Ievan Polkka Trance Remix (Starrodkirby86, Beuchi-chan) \[BeuKirby\]](https://osu.ppy.sh/beatmapsets/10406#fruits/66246)
+  - [Basshunter - Ievan Polkka Trance Remix (Teara) \[BeuKirby\]](https://osu.ppy.sh/beatmapsets/10406#fruits/66246)
   - [DCX - Flying High (DJ Splash Remix) (yeahyeahyeahhh) \[InoSane\]](https://osu.ppy.sh/beatmapsets/22194#fruits/76663)
   - [t+pazolite - Against Gilgamesh (Louis Cyphre) \[Ultimate Illusion\]](https://osu.ppy.sh/beatmapsets/39816#fruits/126671)
   - [Susumu Hirasawa - SWITCHED-ON LOTUS (Starrodkirby86) \[KIRBY Mix Deluxe\]](https://osu.ppy.sh/beatmapsets/16457#fruits/58970)
-  - [DystopiaGround - AugoEidEs (happy30) \[Aphotic\]](https://osu.ppy.sh/beatmapsets/24611#fruits/97416)
+- Tiebreaker  
+  - **[DystopiaGround - AugoEidEs (happy30) \[Aphotic\]](https://osu.ppy.sh/beatmapsets/24611#fruits/97416)**
 
 ### Quarterfinals
 
-- NoMod
+- NoMod (*to be played in order*)
   - [IOSYS - Accept Bloody Fate (Louis Cyphre) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/37305#fruits/120066)
-  - [goreshit - MATZcore (Ekoro) \[Lolicore\]](https://osu.ppy.sh/beatmapsets/24388#fruits/83975)
-  - [Hyadain - Hyadain no Joujou Yuujou (TV Size) (Saten-San) \[Saten's SELAMAT PAGI!!\]](https://osu.ppy.sh/beatmapsets/32775#fruits/112586)
+  - [goreshit - MATZcore (\_LRJ\_) \[Lolicore\]](https://osu.ppy.sh/beatmapsets/24388#fruits/83975)
+  - [Hyadain - Hyadain no Joujou Yuujou (TV Size) (Weezy) \[Saten's SELAMAT PAGI!!\]](https://osu.ppy.sh/beatmapsets/32775#fruits/112586)
   - [DJ Sharpnel - Exciting Hyper Highspeed Star (xBubu) \[Highspeed\]](https://osu.ppy.sh/beatmapsets/16619#fruits/62589)
-  - [Level 70 Elite Tauren Chieftain - I Am Murloc (CheeseWarlock) \[Northrend\]](https://osu.ppy.sh/beatmapsets/12030#fruits/45433)
+- Tiebreaker  
+  - **[Level 70 Elite Tauren Chieftain - I Am Murloc (CheeseWarlock) \[Northrend\]](https://osu.ppy.sh/beatmapsets/12030#fruits/45433)**
 
-### Round 5
+### Group Stage week 5
```

Comment:
it feels like "week" should be capitalised because it is kind of a proper noun? Very much a nitpick tho

Reply:
don't think so

![image](https://user-images.githubusercontent.com/36564236/106345864-bcd46100-62e5-11eb-8b7f-bc12dc968787.png)